app/src/main/java/co/nimblehq/compose/crypto/ui/components/linechart/GradientLineShader.kt

```diff
@@ -0,0 +1,19 @@
+package co.nimblehq.compose.crypto.ui.components.linechart
+
+import androidx.compose.ui.graphics.Brush
+import androidx.compose.ui.graphics.Canvas
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.Path
+import androidx.compose.ui.graphics.drawscope.DrawScope
+import me.bytebeats.views.charts.line.render.line.ILineShader
+
+data class GradientLineShader(
```

Comment:
You accidentally forgot to remove this redundant class 🤓

Reply:
💩